api/v1beta2/awsmachine_types.go

```diff
@@ -199,6 +216,19 @@ type AWSMachineSpec struct {
 	CapacityReservationID *string `json:"capacityReservationId,omitempty"`
 }
 
+// HostPlacement denotes the placement settings for the instance when tenancy=host.
+type HostPlacement struct {
+	// HostID pins the instance to a sepcific Dedicated Host.
+	// +optional
```

Comment:
isn't a hostID required if I set HostPlacement?

Reply:
why? you might just want to set the affinity.